File: proxy/quickstart/create-service/create-service.6.x.py
# Get the Node helper library from https://twilio.com/docs/libraries/python
from twilio.rest import Client

# Get your Account SID and Auth Token from https://twilio.com/console
account = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
token = "your_auth_token"

client = Client(account, token)

service = client.proxy.services.create(
    unique_name="My Awesome Service",
    callback_url="https://www.example.com/"
)

print(service.sid)

File: proxy/quickstart/create-service/create-service.6.x.rb
# Get the Ruby helper library from https://twilio.com/docs/libraries/ruby
require 'twilio-ruby'

# Get your Account SID and Auth Token from https://twilio.com/console
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']

# Initialize Twilio Client
client = Twilio::REST::Client.new account_sid, auth_token

service = client.proxy.services.create(
  unique_name: 'My Awesome Service',
  callback_url: 'https://www.example.com/'
)

puts service.sid
